core/data/repository/src/main/java/com/chocolatecake/repository/MovieRepositoryImpl.kt

```diff
@@ -175,5 +183,17 @@ class MovieRepositoryImpl @Inject constructor(
         } catch (_: Throwable) {
         }
     }
+
+    override suspend fun getPerson(person_id: Int): PeopleDataEntity {
+        return domainPeopleDetailsMapper.map(wrapApiCall { movieService.getPerson(person_id) })
+    }
+
+    override suspend fun getMoviesByPerson(person_id: Int): List<MovieEntity> {
+        return domainMoviesByPeopleMapper.map( wrapApiCall { movieService.getMoviesByPerson(person_id)}.cast!!)
```

Comment:
```suggestion
val call = wrapApiCall { movieService.getMoviesByPerson(person_id)}.cast?.filterNotNull() ?: emptylist()
        return domainMoviesByPeopleMapper.map(call)
```
filter not null cast do not pass !! in map